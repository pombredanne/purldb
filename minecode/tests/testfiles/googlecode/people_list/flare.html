


<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="https://ssl.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"profileUrl": null, "assetVersionPath": "https://ssl.gstatic.com/codesite/ph/13484908847848575064", "token": null, "projectHomeUrl": "/p/flare-browser", "relativeBaseUrl": "", "assetHostPath": "https://ssl.gstatic.com/codesite/ph", "domainName": null, "projectName": "flare-browser", "loggedInUserEmail": null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>People - 
 flare-browser -
 
 
 A site-specific browser for 37signals&#39; Campfire chat application. - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 #resultstable td, #resultstable th { padding-right:2em }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=https%3A%2F%2Fcode.google.com%2Fp%2Fflare-browser%2Fpeople%2Flist&amp;followup=https%3A%2F%2Fcode.google.com%2Fp%2Fflare-browser%2Fpeople%2Flist" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/flare-browser">
 <a href="/p/flare-browser/">
 
 
 <img src="/p/flare-browser/logo?cct=1341866564"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/flare-browser/"><span itemprop="name">flare-browser</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/flare-browser/"><span itemprop="description">A site-specific browser for 37signals&#39; Campfire chat application.</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/flare-browser/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/flare-browser/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 
 
 <a href="/p/flare-browser/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/flare-browser/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st3">
 <div class="isf">
 <span class="inst1">
 <a href="/p/flare-browser/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/flare-browser/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<script type="text/javascript">
function _showBelow(){}
function _toggleStar(){}
function _rowRolloverOn(){}
function _rowRolloverOff(){}
function _goIssue(){}
function _goFile(){}
</script>

<script type="text/javascript">
 var cancelBubble = false;
</script>
<form method="POST" action="addMembers.do"
 >
<div id="colcontrol">
 <div class="list">
 <div class="pagination">
 
 1 - 6 of 6
 
 </div>
 <b>Project People</b>
 &nbsp; &nbsp;
 
 </div>
 <table cellspacing="0" cellpadding="2" border="0" class="results" id="resultstable" width="100%">
 
 <thead>
 <tr id="headingrow">
 
 
 <th style="white-space:nowrap">Name</th>
 
 <th style="white-space:nowrap">Role</th>
 
 <th style="white-space:nowrap">Duties</th>
 
 <th style="white-space:nowrap">Notes</th>
 
 </tr>
 </thead>
 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105534639569596789966')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105534639569596789966"
 >mattbrin...@gmail.com</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105534639569596789966')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105534639569596789966"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105534639569596789966')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105534639569596789966">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105534639569596789966')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105534639569596789966"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=104109063926475518692')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=104109063926475518692"
 >gera...@gmail.com</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=104109063926475518692')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=104109063926475518692"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=104109063926475518692')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=104109063926475518692">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=104109063926475518692')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=104109063926475518692"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105562036645536730749')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105562036645536730749"
 >rakke...@gmail.com</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105562036645536730749')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105562036645536730749"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105562036645536730749')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105562036645536730749">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=105562036645536730749')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=105562036645536730749"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106898263931419003397')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106898263931419003397"
 >dmen...@gmail.com</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106898263931419003397')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106898263931419003397"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106898263931419003397')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106898263931419003397">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106898263931419003397')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106898263931419003397"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=schall.mark')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=schall.mark"
 >schall.mark</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=schall.mark')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=schall.mark"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=schall.mark')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=schall.mark">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=schall.mark')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=schall.mark"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106653752422995692757')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106653752422995692757"
 >s...@samplusplus.com</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106653752422995692757')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106653752422995692757"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106653752422995692757')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106653752422995692757">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/flare-browser/people/detail?u=106653752422995692757')"
 ><a onclick="cancelBubble=true" href="/p/flare-browser/people/detail?u=106653752422995692757"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 
 </table>
 <div class="list-foot">
 <div class="pagination">
 
 1 - 6 of 6
 
 </div>
 &nbsp;
 </div>
</div>




</form>

<script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/13484908847848575064/js/dit_scripts.js"></script>

 
 
 
 <script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/13484908847848575064/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

