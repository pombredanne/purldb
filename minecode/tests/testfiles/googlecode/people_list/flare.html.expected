([{'detail_url': u'http://code.google.com/u/105534639569596789966',
   'person_id': u'105534639569596789966',
   'role': u'owner',
   'username': u'mattbrin...@gmail.com'},
  {'detail_url': u'http://code.google.com/u/104109063926475518692',
   'person_id': u'104109063926475518692',
   'role': u'committer',
   'username': u'gera...@gmail.com'},
  {'detail_url': u'http://code.google.com/u/105562036645536730749',
   'person_id': u'105562036645536730749',
   'role': u'committer',
   'username': u'rakke...@gmail.com'},
  {'detail_url': u'http://code.google.com/u/106898263931419003397',
   'person_id': u'106898263931419003397',
   'role': u'committer',
   'username': u'dmen...@gmail.com'},
  {'detail_url': u'http://code.google.com/u/schall.mark',
   'person_id': u'schall.mark',
   'role': u'committer',
   'username': u'schall.mark'},
  {'detail_url': u'http://code.google.com/u/106653752422995692757',
   'person_id': u'106653752422995692757',
   'role': u'committer',
   'username': u's...@samplusplus.com'}],
 [])